pkgs/by-name/fl/flashprog/package.nix

```
@@ -12,14 +12,14 @@
 , withGpio ? stdenv.isLinux
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
```

Can you please change this to 

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

`rec` is discouraged, and I don't want to include it more.
Otherwise LGTM.

---

Thanks for pointing that out. Done.